operators/tokenizer/tokenizer_jsconfig.hpp

```diff
@@ -46,6 +46,7 @@ class TokenJsonConfig final {
   ~TokenJsonConfig() {}
   using json = nlohmann::json;
   using json_pointer = nlohmann::json_pointer<std::string>;
+  json added_tokens_decoder;
```

Comment:
this json object may not be valid since most of json object are temporary.

Reply:
good catch! will change to use a shared pointer to avoid this issue.